src/common/src/lib.rs

```diff
@@ -82,6 +82,7 @@ pub use {
 pub mod lru;
 pub mod opts;
 pub mod range;
+pub mod rate_limit;
```

Comment:
Better not to put in `common`, but a new crate

Reply:
BTW. May I ask what crate `common` is supposed to do now?

The `RateLimiter` is used by source, sink, streaming executor, dml, etc. It is actually more widely used than other modules in the `common` crate.

Reply:
It's convenient to put things in `common`, but the problem is we want to avoid `common` being too large since it's the single bottleneck of compilation. And we do want to split some code from there. https://github.com/risingwavelabs/risingwave/issues/19623

IMO the most important aspect is dependency relationship between crates. e.g., `RateLimiter` is depended by only `stream` and `connector` crates. If something is depended by also `expr`, `meta`, then it's perfectly `common`. 🤪